engine/portfolio_manager.go

```diff
@@ -253,20 +253,26 @@ func (m *portfolioManager) getExchangeAccountInfo(exchanges []exchange.IBotExcha
 			}
 			continue
 		}
-		assetTypes := exchanges[x].GetAssetTypes(false) // left as available for now, to sync the full spectrum
-		var exchangeHoldings account.Holdings
+
+		assetTypes := asset.Items{asset.Spot}
+		if exchanges[x].HasAssetTypeAccountSegregation() {
+			// Get entire supported exchange asset types to sync all account
+			// information.
+			assetTypes = exchanges[x].GetAssetTypes(false)
```

Comment:
Having false here will impact things like Kraken, who have separate keys for futures and spot, so it will error every time. Seems like `true` would be more appropriate, but I'm happy to hear your thoughts on it

Reply:
Yeah, that is a good point. For now, I will change this to true and add in some extra commentary.